websites/demo-react-image-next/src/app/load/page.tsx

```diff
@@ -5,19 +5,17 @@ import { Suspense } from "@suspensive/react";
 import Link from "next/link";
 import { Area } from "~/components/uis";
 
-const BoundaryPage = () => {
-  return (
-    <div>
-      <Link href="/">to home</Link>
-      <Area title="Load">
-        <Suspense.CSROnly>
-          <Load src="https://ik.imagekit.io/ikmedia/women-dress-2.jpg">
-            {(loaded) => <img src={loaded.src} />}
-          </Load>
-        </Suspense.CSROnly>
-      </Area>
-    </div>
-  );
-};
+const BoundaryPage = () => (
+  <div>
+    <Link href="/">to home</Link>
+    <Area title="Load">
+      <Suspense.CSROnly>
+        <Load src="https://ik.imagekit.io/ikmedia/women-dress-2.jpg">
+          {(loaded) => <img src={loaded.src} />}
```

Comment:
We're using `fepack/image`, which isn't limited to just Next.js. This suggests that the standard img tag could be more universally applicable for our needs. However, the `@next/next/no-img-element` lint warning discourages its use, potentially conflicting with our project's direction. What are your thoughts on disabling this lint warning to align with our approach?

Reply:
After I thought about this warn, If I can, I want to left it as warn now like now.
I think showing warning is good for people want to see demo that are familiar with normal next eslint. we have to know our issue with next.js users